datasource/common/src/main/java/org/opennms/alec/datasource/common/ImmutableAlarm.java

```diff
@@ -228,7 +274,10 @@ public String toString() {
                 ", inventoryObjectType='" + inventoryObjectType + '\'' +
                 ", summary='" + summary + '\'' +
                 ", description='" + description + '\'' +
-                ", nodeId=" + nodeId +
+                ", nodeId=" + nodeId + '\'' +
```

Comment:
I think you are missing some ' after the = for nodeId, nodeLabel, nodeLocation and reduction key

Reply:
It was missing for other fields too, I regenerated the toString with StringJoiner